From ca74ccbcd767b9377dc01553091cef9726708525 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Tue, 24 Mar 2015 14:01:54 +0100
Subject: [PATCH] Revert "display-vnc: fix zoom-level set by command line"

This reverts commit 68148e1bd1a47ff370c78e2569a57ae0f3d8a400.

The commit in question was causing a regression about a tiny window
being displayed when opening a vnc guest.
The issue only happens with gtk-vnc/gtk2/virt-viewer versions present
on RHEL-6.7 and does not affect (gtk2 versions of) the upstream
packages, being this the reason to keep the patch downstream only.

Resolves: rhbz#1201679
---
 src/virt-viewer-display-vnc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/virt-viewer-display-vnc.c b/src/virt-viewer-display-vnc.c
index ecda431..4b7243b 100644
--- a/src/virt-viewer-display-vnc.c
+++ b/src/virt-viewer-display-vnc.c
@@ -118,7 +118,6 @@ virt_viewer_display_vnc_initialized(VncDisplay *vnc G_GNUC_UNUSED,
 {
     virt_viewer_display_set_show_hint(display,
                                       VIRT_VIEWER_DISPLAY_SHOW_HINT_READY, TRUE);
-    g_signal_emit_by_name(display, "display-desktop-resize");
 }
 
 static void
